drivers/google/google.go

```diff
@@ -216,28 +216,39 @@ func (d *Driver) GetState() (state.State, error) {
 	return state.None, nil
 }
 
-// Start creates a GCE instance and attaches it to the existing disk.
+// Start starts an existing GCE instance or create an instance with an existing disk.
 func (d *Driver) Start() error {
 	c, err := newComputeUtil(d)
 	if err != nil {
 		return err
 	}
-	if err = c.createInstance(d); err != nil {
-		return err
+
+	instance, _ := c.instance()
```

Comment:
You are right. I pasted the code from GetState(). What really matter is if there's an instance or not. But this could swallow other kind of errors. I'll take a look.